lib/spec/openapi/utils.js

```diff
@@ -407,6 +409,13 @@ function prepareOpenapiMethod (schema, ref, openapiObject) {
     }
   }
 
+  // If there is no schema or schema.params, we need to generate them
+  if ((!schema || !schema.params) && hasParams(url)) {
+    const schema = generateParamsSchema(url)
+    resolveCommonParams('path', parameters, schema.params, ref, openapiObject.definitions)
```

Comment:
It was confusing me as I though we were overwriting the input `schema`

```suggestion
    const schemaGenerated = generateParamsSchema(url)
    resolveCommonParams('path', parameters, schemaGenerated.params, ref, openapiObject.definitions)
```